src/LessMsi.Cli/eExtractionMode.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace LessMsi.Cli
+{
+    public enum eExtractionMode
+    {
+        None = 0,
+        RegularExtraction = 1,
```

Comment:
Please add a `///` comment to each of these to document them. Maybe also rename `RegularExtraction` to a more descriptive name like `PreserveDirectories` or something